models/gc_stoic_baseline/meta.json

```diff
@@ -0,0 +1,125 @@
+      "code": "2022-01-04",
+      "pub": "2023-06-25"
+    },
+    "cite": "Luuk H. Boulogne, Julian Lorenz, Daniel Kienzle, Robin Schon, Katja Ludwig, Rainer Lienhart, Simon Jegou, Guang Li, Cong Chen, Qi Wang, Derik Shi, Mayug Maniparambil, Dominik Muller, Silvan Mertes, Niklas Schroter, Fabio Hellmann, Miriam Elia, Ine Dirks, Matias Nicolas Bossa, Abel Diaz Berenguer, Tanmoy Mukherjee, Jef Vandemeulebroucke, Hichem Sahli, Nikos Deligiannis, Panagiotis Gonidakis, Ngoc Dung Huynh, Imran Razzak, Reda Bouadjenek, Mario Verdicchio, Pasquale Borrelli, Marco Aiello, James A. Meakin, Alexander Lemm, Christoph Russ, Razvan Ionasec, Nikos Paragios, Bram van Ginneken, Marie-Pierre Revel Dubois, The STOIC2021 COVID-19 AI challenge: applying reusable training methodologies to private data, arXiv:2306.10484, 2023, https://arxiv.org/abs/2306.10484v2",
+    "license": {
+      "code": "MIT license",
+      "weights": "MIT license"
+    },
+    "publications": [
+      {
+        "title": "The STOIC2021 COVID-19 AI challenge: applying reusable training methodologies to private data",
+        "uri": "https://arxiv.org/pdf/2306.10484v2.pdf"
+      }
+    ],
+    "github": "https://github.com/luukboulogne/stoic2021-baseline",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Use",
+      "text": "The algorithm processes a CT scan of a suspected COVID-19 patient and produces the probability of a COVID-19 infection and a probability of severe outcome (death or intubation within one month).",
+      "references": [],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Analysis",
+      "text": "A single I3D model, initialized with publicly available weights trained for RGB video classification, was trained to estimate both COVID-19 presence and severity. The model was trained on all training data for 40 epochs using the AdamW optimizer with a learning rate of 10, momentum parameters β1 = 0.9, β2 = 0.99, and a weight decay of 0.01. Data augmentation was employed in the form of zoom, rotation, translation, and adding gaussian noise. Patient age and sex information were not incorporated as input to the model.",
+      "references": [],
+      "tables": []
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The entire STOIC dataset",
+      "references": [
+        {
+          "label": "Stoic project",
+          "uri": "https://pubs.rsna.org/doi/10.1148/radiol.2021210384"
+        }
+      ],
+      "tables": []
+    },
+    "training": {
+      "title": "Training data",
+      "text": " A publicly available subset of the STOIC dataset",
+      "references": [
+        {
+          "label": "Stoic project",
+          "uri": "https://pubs.rsna.org/doi/10.1148/radiol.2021210384"
+        }
+      ],
+      "tables": []
+    },
+    "ethics": {
+      "title": "",
+      "text": "",
+      "references": [],
+      "tables": []
+    },
+    "limitations": {
```

Comment:
> Lastly, the maximum obtainable performance is limited by imperfections in the COVID-19 severity and presence labels. Death at one month follow-up could have resulted from any cause. RT-PCR is an imper- fect ground truth for infection. For the STOIC study, 39% of initially negative RT-PCR tests were found to be posi- tive when repeated in patients with typical clinical signs of COVID-19 [67].

_from https://arxiv.org/abs/2306.10484, p. 18_

The Model Card section can indeed be used more for technical constraints (e.g. specific requirements or input data a model needs), but in the general sense of MHub this should rarely be necessary anyway. So I'm not asking to copy/paste the limitations section from teh original publication here. However, I think the lack of any details under Training, Evaluation and Limitations gives a wrong impression. This could very well be mentioned under Evaluation to give the user a more transparent idea of the expected model performance.

Reply:
We'll add it under the limitations of the model.